### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by alex-hse-repository**

### CODE REVIEW

File: tests/test_transforms/test_impute_transform.py [hunk 1]
```diff
@@ -50,12 +50,11 @@ def test_wrong_init_one_segment():
         _ = _OneSegmentTimeSeriesImputerTransform(strategy="wrong_strategy")
 
 
-# TODO: fix the test after changing behaviour of PerSegmentWrapper to fail in __init__
 @pytest.mark.xfail
```
Comment (Mr-Geekman) [inline tests/test_transforms/test_impute_transform.py:53]: This line should be deleted, it shows that this test may fail and pytest ignores the result.

**Comment by Mr-Geekman** [comment]
My be we should also remove `finder` part from `SpecialDaysTransform` files and rename class file from `special_day_finder.py` to `special_days` and test file from `test_special_days_finder_transform` to `test_special_days_transform`.

**Comment by julia-shenshina** [comment]
> My be we should also remove `finder` part from `SpecialDaysTransform` files and rename class file from `special_day_finder` to `special_days` and test file from `test_special_days_finder_transform` to `test_special_days_transform`.

➕ 
`_finder` part is a legacy that should be removed

File: tests/test_transforms/test_impute_transform.py (hunk 1 above)
Comment (alex-hse-repository) [inline tests/test_transforms/test_impute_transform.py:53]:
```suggestion
```

Comment (alex-hse-repository) [reply tests/test_transforms/test_impute_transform.py]: Done

**Review by Mr-Geekman — COMMENTED** [review]
The merge conflict should be solved, then I'll approve.

File: etna/transforms/scalers.py [hunk 2]
```diff
@@ -26,7 +25,6 @@ def __init__(
     ):
         """
         Init StandardScalerPreprocess.
-
```
Comment (Mr-Geekman) [inline etna/transforms/scalers.py:29]: Don't remove this lines, leave this file in sync with master.

File: etna/transforms/sklearn.py [hunk 3]
```diff
@@ -37,12 +36,10 @@ def __init__(
     def fit(self, df: pd.DataFrame) -> "SklearnTransform":
         """
         Fit transformer with data from df.
-
```
Comment (Mr-Geekman) [inline etna/transforms/sklearn.py:40]: Don't remove this lines, leave this file in sync with master.

File: etna/metrics/base.py [hunk 4]
```diff
@@ -10,12 +11,19 @@
...
 
     macro = "macro"
     per_segment = "per-segment"
 
+    @classmethod
+    def _missing_(cls, value):
+        raise NotImplementedError(
+            "%r is not a valid %s. Only %s aggregation allowed"
+            % (value, cls.__name__, ", ".join([repr(m.value) for m in cls]))
+        )
```
Comment (iKintosh) [inline etna/metrics/base.py:25]: Please rewrite it as f-string

**Pull request closed without merging**